src/hatti/constants.cljs

```diff
@@ -4,3 +4,4 @@
 (def _rank "_rank")
 (def _submission_time "_submission_time")
 (def _submitted_by "_submitted_by")
+(def *mapping-threshold* 10000)
```

Comment:
I think it should be. Or we should expose it differently as a configurable option. Different Hatti users may have different thresholds.